package-lock.json

```diff
@@ -1,7917 +1,51 @@
 {
   "name": "wagtail.org",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
note: I think you might be using an old version of node here.

Reply:
Thanks!